tests/event-parser-test.js

```diff
@@ -2,10 +2,81 @@
 
 const assert = require('assert');
 
-const { EventParser, EventParserError } = require('./../lib');
+const sandbox = require('sinon').createSandbox();
+
+const Package = require('./../index');
+
+const { EventParser, ModelEvent } = require('./../lib');
 
 describe('EventParser', () => {
+	const message = {
+		client: 'core',
+		entity: 'test',
+		event: 'testing'
+	};
+
+	const getEvent = {
+		client: 'core',
+		entity: 'test',
+		event: 'testing',
+		subscribers: [
+			{
+				source: 'shipping',
+				client: 'core/client',
+				entity: 'order',
+				event: 'create'
+			}
+		]
+	};
+
+	afterEach(() => {
+		sandbox.restore();
+	});
+
+	it('should return and instance of event parser', () => {
```

Comment:
Mayúscula a todos los `it()` porfa..
```suggestion
	it('Should return and instance of event parser', () => {
```